frontend/src/containers/Search/SearchPage.js

```diff
@@ -0,0 +1,62 @@
+import React, { Component } from 'react';
+import { StyleSheet, css } from 'aphrodite';
+//import SearchInput from './SearchInput';
+import SubmitInput from './SubmitInput';
+import Form from './Form';
+
+export default class SearchPage extends Component {
+  getInputValue = () => {
+    return this.input.value;
+  }
+
+  handleKeyUp = (e) => {
+    if (e.keyCode === 13) {
+      this.handleSubmit();
+    }
+  }
+
+  handleSubmit = () => {
+    const postCode = this.getInputValue();
+    this.props.history.push(`search?postCode=${postCode}`);
+  }
+
+  render () {
+    return (
+      <div className={css(styles.container)}>
+        <Form onSubmit={this.handleSubmit}>
+          <input type="search"
+                 className={css(styles.input)}
```

Comment:
indent two spaces? Oh I see you've lined up the attribs, is that normal? Looks good, I can do that :)

Reply:
I looked good to my eyes ;)